plugin/build.gradle

```diff
@@ -27,7 +27,7 @@ dependencies {
     compileOnly gradleApi()
 
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation "com.github.kittinunf.fuel:fuel:1.10.0"
+    implementation "com.github.kittinunf.fuel:fuel:1.16.0"
```

Comment:
Is there a way to ensure that the one-sky task completes normally after this? 

Maybe running: 

`./gradlew :ui-commons:uploadTranslationAndMarkAsDeprecated`?

Reply:
good point. I just check by pulling the snapshot 
```
    implementation 'com.github.cookpad:onesky-gradle-plugin:hw~update-agp-version-SNAPSHOT'
```

`./gradlew downloadTranslation --stacktrace` which worked 

Anway I am not going to change the version yet in global-android, it will still pick up the latest git tag version  , when I do, I will do the test with the snapshot or new tag version and the bump the version.